alonzo/impl/test/lib/Test/Cardano/Ledger/Alonzo/Serialisation/Generators.hs

```diff
@@ -192,8 +192,14 @@ instance Arbitrary Language where
 instance Arbitrary Prices where
   arbitrary = Prices <$> arbitrary <*> arbitrary
 
+newtype AlphaString = AlphaString {unAlphaString :: String}
+  deriving (Show, Eq, Ord)
+
+instance Arbitrary AlphaString where
+  arbitrary = AlphaString <$> listOf (elements ['a' .. 'z'])
+
```

Comment:
Couldn't you just write a function  genString :: Gen String, and use that, rather than making a one time use newtype?

Reply:
I wanted to use the map arbitrary instance, which would call `arbitrary` for `Text` on all the keys, which we don't want.